.travis.yml

```diff
@@ -6,11 +6,27 @@ env:
+  - secure: "b7f77aPg0gOJ5nHcKBiR6Pu7+V2Mq4TZ+f6bcmCmyNh7jpNwZd0PN2XRKaGuautFfGMd7RmkKsdfyWDeqm3J+YR6LzEqwaAULfzvrlQNId8AVFfrke/uoNwNOUczX5L9h5zpLK63AzUXIO8KwVO4d8xBV8wtcW+AqjC5q79Wq2ZfETELN7l+0ZtaAwjgdDtMufhcRvZ8TIw1VTM6D6fclumf7qaoDZF3BSnS8Ar+m9yfuMDoDckt+dT2bFEBN+SK3/IOJdHxlzbMsEQSDrz6c1HA+rOkiEYDnO7wfJguyTJp3LURBDlt2ScMsL2LGXiV0Y0rK26lvizGjlvgSfQmK7dG1Xtv8fPDS++K+E2Mpa8jiCb4ZxAxxJs7j4KK+hBpsjTBglE6k7Zb0/IeLYaf05SKwVz9+e5zrlaCBQ5vul9XQikLy1fHw4jXIMqjzaHlWt5SqFX2p+yK31xs76aCNNy/RT/QKjEh6iEwsN3QEzAgHqecjehArbSGod5Q1t0rNspsssUj8li3iFRu3cmoBSlSpsqtndu7i0FP5lIRHbE4p/Xq1Gw4XNM2FCD+1bLiyJQfkSWNeghIA189cECvLDyjyiaNNg3bZHMKa1ocjqRzOF1Hj9JMksAxSyxD6zYwXvEvl8ku6gog1MvEfTgiP+kQAAnnOOUjvq5IjlU2s64="
+  - secure: "Shfykzt3i9J5ShuzzPS9A9YapSYE9ggyUckybq11KS7mfv4RegaWHQTIkohUbdP9covts9+m+vaBMUqHKpTqCGtZZv5TEoDCypohXujc9ml1Kk3qc3N/6FYrpt8qirplWRMuicCpvmIRzrQ22kpNupkE+7SkjB5T9JzKeyFZHDX5+fTFF2O41VhUvdZDBSPl+UtlCwruSzlBxHc7ZQZswxQgEy90SX5mJspH+QANRPE+hMpMVPUgrExnC4N7gVFCqnIOcSgzk4u/RbMm1Fye6a6+h1mU6Rmm9e/86oZ0LU1vTdj6TFYOapT0WrPw8W7X1rTC+29YnHg3ikvoHqQo0+uWhRP5Wb6BHtq12xfYQVlGgy+kRV2pfB7AEPlOp9jCQeAWx0iu0Of1QCuWyRQKM89JUErc4Ee2aOWJPr45EZaUPWzObfRET8QJIRxiX21520KiIc2MOd6dOzDba5SHJk3ZcvTHwC00ZfbG2/dmEkw8fgQwQHfG+YszfkO8LuCaitRFU19+q/qf4IfMJNMZUVtok4uuW63jkurSXDR4ub8mbyHH42Rj8+x1/NuuCq54cTw0lF8ya3ZNIp2StMqjnK1m5XUXoqVbB1PCry9MlggNKnqEW5wFTNuS6pyrW8GanfeSITI+36StpGGwlHNhSFg0BzR78Hc059kxy8YYKqs="
+  - secure: "d4NtPyHgynqStDDrE/vBG3BWAM4fJn4x5f7fnfwLmanooYrtC2zd7EAIfTBTnF0e3rgJDJt+3MeLjIlOlBxJddDboUtcQkkEwFayhwnHEvxCnRyPS9oLMmRsmqiSULCA9U5sFtilVRHGDejCWKpoWUagVvzLjga1MlL/dweUR24046VjD0m6javl0i0a0GED9RxARGJo0vlj31jOXJu9RDi2w+0ppkCOxcYeH9+Dzt02yhrpX0s9DTskZzDw4JdUym7W6fA0e06yK9MEOshMgfQOokE1qt3X6fJFKSvds1dnXOrL6+2JyvNyiuc/JRDujP6+Y+whJhhg2hKF91AU/udenT8VdOPQEY94yK3zVtebjjwv08iIZdJ7pGOp76qb3EGZV7rN2kjzum7Ay7sFhmtoTcTPrO/1oIjf4eCxIOPWMr21njPmsUrufBxAjEMuB8M/HR6y9ZPhm+wpHbUU9wD0YhNLS1j9i0sCEaUF6fJMDcPDIHBmvsfGGIN7VTRaMwvgfeFftDakH2Z8Ore8ZZ6j/lxytcbXu4n/pl/JJUXbF8+bw6GZrAguLPrh0qbqQTPJXSzHKBh53+1nTh5s63PDXZ4XXgT9v/ghMyW4AyxNosRvmqjrPAihQiAjSPqKebP1gBJgPJH6WyNLJpnKBirewgulus1t8Q0EVfWSUS8="
 
 script:
 - docker build --tag todo-app:test --target test .
 - docker run todo-app:test todo_app/Tests/test_items.py
 - docker run --env-file .env.test todo-app:test todo_app/Tests/test_app.py
 - docker run --env TRELLO_API_BOARD_ID --env TRELLO_API_TOKEN --env TRELLO_API_KEY  todo-app:test todo_app/Selenium_Tests/test_Selenium.py
+- echo $DOCKER_PASSWORD | docker login --username $DOCKER_USER --password-stdin
 
+before_deploy:
+- heroku container:login
+- docker build --target production --tag $DOCKER_USER/todo-app:latest --target production .
+- docker push $DOCKER_USER/todo-app:latest
+- docker tag $DOCKER_USER/todo-app:latest registry.heroku.com/todoapp-jb/web  
```

Comment:
You could also just create both tags in the first place with e.g. `docker build --tag tag-one --tag tag-two .`